README.md

```diff
@@ -1 +1,50 @@
-# Embedding-Images-and-Video
\ No newline at end of file
+# Embedding Images and Video
+
+> This page is a clone of a youtube page made for the Milestone 1 project of Microverse.
+
+![screenshot](media/screenshot.JPG)
+
+Additional description about the project and its features.
```

Comment:
Make sure that the project description is in the README before submitting it for code review.